test/utils/toolpadApi.ts

```diff
@@ -0,0 +1,40 @@
+import './fetchPolyfill';
+import { createRpcClient } from '../../packages/toolpad-app/src/rpcClient';
+import type { ServerDefinition } from '../../packages/toolpad-app/pages/api/rpc';
+import generateId from './generateId';
+import type { CreateAppOptions } from '../../packages/toolpad-app/src/server/data';
+
+export function getApi(endpoint: string) {
+  return createRpcClient<ServerDefinition>(endpoint);
+}
+
+interface CreateApplicationParams {
+  name?: string;
+  appTemplateId?: string;
```

Comment:
I created an `AppTemplateId` type that maybe could be imported to use here?

Reply:
I'm not sure a union type makes sense for this, it feels like data to me.

Reply:
That's fair, just mentioning it.